Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM golang:1.5.1
```

Comment:
This should inherit from https://github.com/elastic/libbeat/blob/master/Dockerfile